requirements-dev.txt

```diff
@@ -0,0 +1,6 @@
+-r requirements.txt
+mock
+pep8==1.6.2
```

Comment:
Note that in `.travis.yml`, we're now using this `requirements-dev.txt` so local and Travis will have the same versions.

I can see your point about being less strict with the version requirement, but consider that 1.7.0 might introduce breaking changes again, and then we're left with the same problem again.